WordPress/UITests/Tests/LoginTests.swift

```diff
@@ -10,28 +10,26 @@ class LoginTests: XCTestCase {
 
     override func tearDownWithError() throws {
         takeScreenshotOfFailedTest()
-        removeApp()
     }
 
     // Unified email login/out
     func testWPcomLoginLogout() throws {
-        let prologueScreen = try PrologueScreen().selectContinue()
+        try PrologueScreen().selectContinue()
```

Comment:
this change is not really necessary for this change but i decided to take advantage to standardize this since there was 1 test in this file that had to be updated to work with this new solution